test/unit/object/test_events.py

```diff
@@ -181,6 +182,18 @@ def mock_event():
 
 
 @pytest.fixture()
+def mock_event_object(mock_box_session):
+    return Event(mock_box_session, "f82c3ba03e41f7e8a7608363cc6c0390183c3f83", {
```

Comment:
This mock event object fixture should depend on the mock_event fixture (since it looks like it copies the json) and use that json blob to construct the event rather than copying.  This will make it easier to maintain the code; fewer places to change if the event json needs to be updated.